doc-content/jbpm-docs/src/main/asciidoc/Persistence/transaction-register-con.adoc

```diff
@@ -0,0 +1,86 @@
+[id='transaction-register-con_{context}']
+= Registering a transaction manager
```

Comment:
Again, either need to make this a bona fide proc or change the title to be for the con, such as "Transaction manager registration"